x-pack/plugins/actions/server/lib/create_action_event_log_record_object.ts

```diff
@@ -68,6 +79,10 @@ export function createActionEventLogRecordObject(params: CreateActionEventLogRec
       ...(task ? { task: { scheduled: task.scheduled, schedule_delay: task.scheduleDelay } } : {}),
     },
     ...(message ? { message } : {}),
+    action: {
+      name,
+      uuid: actionExecutionId,
```

Comment:
ah ... there's the uuid!  Yeah, let's duplicate the whole structure, since we may add additional bits under `execution`, and the parallelism is nice and familiar.  So it would be `kibana.action.execution.uuid` I think ...

Reply:
Resolved in this commit f933a09f2240f33b542881dcb715e6c7b4a645dc